.github/workflows/release.yml

```diff
@@ -154,7 +170,7 @@ jobs:
             mv ./target/release/${binary}${{ env.extension }} ${{ env.TMP_BUILD_DIR }}/${binary}${{ env.extension }}
           done
 
-          tar -cvzf ./tmp/${{ env.artifact_name }}.tgz -C ${{ env.TMP_BUILD_DIR }} .
+          tar -cvzf ./tmp/${{ needs.generate-tag.outputs.artifact_name }}.tgz -C ${{ env.TMP_BUILD_DIR }} .
```

Comment:
Does this magically take the right output according to matching matrix?

Reply:
I think so